recipe/meta.yaml

```diff
@@ -1,40 +1,41 @@
-{% set version = "2.10.0" %}
+{% set version = "2.11.0" %}
 
 package:
   name: distributed
   version: "{{ version }}"
 
 source:
   url: https://pypi.io/packages/source/d/distributed/distributed-{{ version }}.tar.gz
-  sha256: 2f8cca741a20f776929cbad3545f2df64cf60207fb21f774ef24aad6f6589e8b
+  sha256: 8252e9e6387db3862c9b56b5e028b5a592fa82dcbe100bffc34be58ccd74e01c
 
 build:
-  number: 1
+  number: 0
   entry_points:
     - dask-scheduler=distributed.cli.dask_scheduler:go
     - dask-ssh=distributed.cli.dask_ssh:go
     - dask-worker=distributed.cli.dask_worker:go
   script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
-  noarch: python
+  skip: True  # [py<36]
 
 requirements:
   host:
-    - python >=3.6
+    - python
     - pip
 
   run:
-    - python >=3.6
+    - python
     - click >=6.6
     - cloudpickle >=0.2.2
     - cytoolz >=0.7.4
     - dask-core >=2.9.0
-    - msgpack-python <1.0.0
+    - msgpack-python >=0.6.0
     - psutil >=5.0
     - pyyaml
     - sortedcontainers !=2.0.0,!=2.0.1
-    - tblib
+    - tblib >=1.6.0
     - toolz >=0.7.4
-    - tornado >=5
+    - tornado >=5  # [py<38]
+    - tornado >=6.0.3  # [py>=38]
```

Comment:
Is it really necessary to make this distinction and lose `noarch: python`?

Reply:
AIUI yes. @jrbourbeau, care to comment? 🙂

Reply:
I meant instead of just,
```yaml
- tornado >=6.0.3
```

Reply:
I don't think we should be altering upstream requirements to preserve `noarch: python`. If they added them, there must be a reason and we should respect that.

Reply:
There are some systems that don't run well on tornado 6, so we want to support older versions of tornado where possible (xref https://github.com/dask/distributed/pull/3249#issuecomment-574418510). Eventually we will want to move back to noarch

Reply:
Are there tornado packages <6.0.3 for Python 3.8 available in `conda-forge`?

Reply:
To answer my own question, yes there are 4.5.3 and 5.1.1 packages in conda-forge.